kolibri/core/auth/management/utils.py

```diff
@@ -227,11 +228,16 @@ def get_client_and_server_certs(
                 username = input("Please enter username: ")
                 password = getpass.getpass("Please enter password: ")
 
+        # add facility so `FacilityUserBackend` can validate
+        userargs = {
+            FacilityUser.USERNAME_FIELD: username,
+            FACILITY_CREDENTIAL_NAME: dataset_id,
```

Comment:
This is being passed as the facility id, but is the `dataset_id` - is this not `facility.dataset_id` or is the variable name here just confusing?